### PR TITLE
feat: integrate vee-validate

### PR DESCRIPTION
**Pull request by dreadderino1987**
some change in ts code and integrate typescript finaly to actual version

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,68 @@
 # music-app
 
-## Ein webbasierter selbst-gehosteter Musikplayer
```
Comment (NonoSikk) [inline README.md:3]: deutsch wieder rein

File: src/stores/storeAuth.ts [hunk 2]
```diff
@@ -29,7 +30,7 @@ export const useStoreAuth = defineStore("storeAuth", {
         }
       });
     },
-    registerUser(credentials) {
+    registerUser(credentials: any) {
```
Comment (NonoSikk) [inline src/stores/storeAuth.ts:33]: type any?

**Review by NonoSikk — APPROVED** [review]

**Comment by NonoSikk** [comment]
Great job!

**Comment by dreadderino1987** [comment]
previewed by answer quetion with Nina (alumni bootcamp)

**Pull request merged**